Sources/SecureXPC/Server/XPCServer.swift

```diff
@@ -137,11 +149,21 @@ public class XPCServer {
     /// - Throws: If this route has already been registered.
     public func registerRoute(_ route: XPCRouteWithoutMessageWithoutReply,
                               handler: @escaping () throws -> Void) throws {
```

Comment:
In retrospect, I don't think throwing here was a good idea (introduced in #28).

These errors can't reasonably recovered:

1. "Do nothing" => the route doesn't act the way you expect
2. "Name it something different" => makes it useless to the callers that expect a given name

Fatal erroring here might be more appropriate, so that route-registration doesn't need a `try` (removing `try` for brevity is a non-goal for me, but in some cases an unexpected `try` raises a suspicion error was thrown that can't really be caught)

Reply:
Let me think about this. If I make this change, will do so as part of a separate PR.